inst/pkgdown/_pkgdown.yml

```diff
@@ -8,3 +8,23 @@ template:
       <img src="https://github.com/insightsengineering/hex-stickers/blob/main/PNG/nest.png?raw=true" alt="NEST" style="height: 30px;">
       <a href="https://pharmaverse.org/"><img src="https://github.com/insightsengineering/hex-stickers/blob/main/PNG/pharmaverse.png?raw=true" alt="pharmaverse" style="height: 30px;"></a>
       </div>
+  includes:
+    in_header: |
+       <script>
+         // Define dataLayer and the gtag function.
+         window.dataLayer = window.dataLayer || [];
+         function gtag(){dataLayer.push(arguments);}
+         // Default ad_storage to 'denied' as a placeholder
+         // Determine actual values based on your own requirements
+         gtag('consent', 'default', {
+           'ad_storage': 'denied'
+         });
```

Comment:
@dinakar29 
copy-pasted from the reference you provided
do you have any ideas for meaningful defaults?